app/services/bulk_upload/sales/year2022/row_parser.rb

```diff
@@ -944,4 +946,21 @@ def validate_relevant_collection_window
       errors.add(:field_4, I18n.t("validations.date.outside_collection_window"))
     end
   end
+
+  def validate_incomplete_soft_validations
```

Comment:
Are these validations identical?

Reply:
Maybe. I'm following a pattern we're using for all the row parser validations, which is currently defining them individually for each year/type